### PR TITLE
`azurerm_spring_cloud_certificate` - fix testcase

### PR DESCRIPTION
**Pull request by ms-henglu**
image

### CODE REVIEW

File: internal/services/springcloud/spring_cloud_certificate_resource.go [hunk 1]
```diff
@@ -105,7 +106,7 @@ func resourceSpringCloudCertificateCreate(d *pluginsdk.ResourceData, meta interf
 			return err
 		}
 		cert.Properties = &appplatform.KeyVaultCertificateProperties{
-			VaultURI:         &keyVaultCertificateId.KeyVaultBaseUrl,
+			VaultURI:         utils.String(strings.TrimSuffix(keyVaultCertificateId.KeyVaultBaseUrl, "/")),
```
Comment (katbyte) [inline internal/services/springcloud/spring_cloud_certificate_resource.go:109]: could you explain this change?

**Review by katbyte — CHANGES_REQUESTED** [review]

File: internal/services/springcloud/spring_cloud_certificate_resource.go (hunk 1 above)
Comment (ms-henglu) [reply internal/services/springcloud/spring_cloud_certificate_resource.go:109]: Yes, sorry that I forgot to add it in the description.

The backend api will reject if vault uri has a suffix `/`

Comment (katbyte) [reply internal/services/springcloud/spring_cloud_certificate_resource.go:109]: and the stray `/` is coming from where?

**Review by katbyte — CHANGES_REQUESTED** [review]

File: internal/services/springcloud/spring_cloud_certificate_resource.go (hunk 1 above)
Comment (ms-henglu) [reply internal/services/springcloud/spring_cloud_certificate_resource.go:109]: It was added here: https://github.com/hashicorp/terraform-provider-azurerm/blob/main/internal/services/keyvault/parse/nested_item.go#L107

**Review by katbyte — APPROVED** [review]
LGTM 🗝️

**Pull request merged**